tests/integration/kubernetes/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 Microsoft Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+set_runtime_class() {
+    sed -i -e "s|runtimeClassName: kata|runtimeClassName: kata-${KATA_HYPERVISOR}|" runtimeclass_workloads/*.yaml
+}
+
+set_runtime_class
```

Comment:
Nit: could you move this call to a `main()` method then `main $@`? This is a "standard" that we try hard for the scripts to follow. @jodh-intel and I will be pretty happy with that change :)

Reply:
Done